Gemfile

```diff
@@ -3,4 +3,5 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in ibm_power_hmc.gemspec
 gemspec
 
+gem "manageiq-style", "~> 1.3.1"
```

Comment:
@agrare I just included `manageiq-style` in the Gemfile so we'll get whatever `rubocop` version is defined in it's gemspec. Are there any problems with that approach?